src/Notification/Notification.js

```diff
@@ -210,16 +210,27 @@ module.exports = kind(
 				this.release();
 			}
 		}
-
-		if (!this.showing) {
-			Spotlight.unspot();
-		}
 	},
 
 	/**
 	* @private
 	*/
-	shown: function (sender, ev) {
+	showingChanged: kind.inherit(function (sup) {
+		return function (sender, ev) {
+			if (this.showing && this.showingTransitioning) {
+				// ENYO-3382: hide animation is not finished, call popup hide directly before show again
+				this.showing = !this.showing;
+				Popup.prototype.showingChanged.apply(this, arguments);
+				this.showing = !this.showing;
+			}
+			sup.apply(this, arguments);
```

Comment:
You mean remove this.showing = !this.showing; ?
I added these lines because popop.showingchanged with opposite showing value should be called before sup call.